CLI command get with no hash cause panic

Entering the `get` CLI command with no hash into CLI is causing panic.

**Example:**

```
>>> get
```

Give output

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
d7024e/kademlia/network/routing.NewKademliaID({0xc001240144?, 0x6?})
        C:/Users/Rickard/Documents/GitHub/d7024e/kademlia/network/routing/kademliaid.go:26 +0x9f
d7024e/kademlia.(*Kademlia).LookupData(0xc00052bda0, {0xc001240144, 0x0})
        C:/Users/Rickard/Documents/GitHub/d7024e/kademlia/kademlia.go:102 +0x38
d7024e/cli/commands.GetObjectByHash({0x71c0a8?, 0xc00052bda0?}, {0xc001240144?, 0x1?})
        C:/Users/Rickard/Documents/GitHub/d7024e/cli/commands/get.go:10 +0x62
d7024e/cli.(*Cli).Open(0xc0000f7f00, 0x1)
        C:/Users/Rickard/Documents/GitHub/d7024e/cli/cli.go:65 +0x55a
main.start({0x6e7feb, 0xe}, 0x1f72, 0x1)
        C:/Users/Rickard/Documents/GitHub/d7024e/launch.go:52 +0x294
main.main()
        C:/Users/Rickard/Documents/GitHub/d7024e/launch.go:34 +0xb1
```

--- a/cli/commands/get.go
+++ b/cli/commands/get.go
@@ -6,6 +6,9 @@ import (
 )
 
 func GetObjectByHash(context kademlia.IKademlia, args string) (string, error) {
+	if args == "" {
+		return "", errors.New("expected 1 argument, but got 0")
+	}
 	cleanHash := RemoveDoubleQuotes(args)
 	value, _ := context.LookupData(cleanHash)
 	if value != nil {

--- a/cli/commands/put.go
+++ b/cli/commands/put.go
@@ -2,9 +2,13 @@ package commands
 
 import (
 	"d7024e/kademlia"
+	"errors"
 )
 
 func PutObjectInStore(context kademlia.IKademlia, args string) (string, error) {
+	if args == "" {
+		return "", errors.New("expected 1 argument, but got 0")
+	}
 	cleanContent := RemoveDoubleQuotes(args)
 	dataToSend := []byte(cleanContent)
 

--- a/kademlia/kademlia.go
+++ b/kademlia/kademlia.go
@@ -99,18 +99,13 @@ func (kademlia *Kademlia) lookupContactAux(targetID *routing.KademliaID, contact
 
 // send lookup message to closest nodes
 func (kademlia *Kademlia) LookupData(hash string) ([]byte, *routing.Contact) {
-	kademliaIdFromHash := routing.NewKademliaID(hash)
-	contacts := kademlia.LookupContact(kademliaIdFromHash)
+	contacts := kademlia.LookupContact(kademlia.me.ID)
 
-	for _, contact := range contacts { // for each of the <=5 contacts found...
-		//fmt.Println(" trying to find data on node ", contact.ID)
-
-		//kademlia.Network.SendLookup(&contact, hash) //send FindLocally to each
+	for _, contact := range contacts {
 
 		valuechannel := make(chan string, 1)
-		go rpc.SendLookupMessage(kademlia.network, &contact, hash, valuechannel) //send FindLocally to each
+		go rpc.SendLookupMessage(kademlia.network, &contact, hash, valuechannel)
 		value := <-valuechannel
-		//fmt.Printf("Recieved value %v from node %v", value, contact.String())
 		if value != "" {
 			return []byte(value), &contact
 		}

